mastodon/src/main/java/org/joinmastodon/android/fragments/ComposeFragment.java

```diff
@@ -1056,6 +1064,12 @@ public boolean onMenuItemClick(MenuItem item){
 		menu.show();
 	}
 
+	private boolean onVisibilityLongClick(View v) {
```

Comment:
Why isn't this `void` as the `onVisibilityClick()`?

Reply:
`LongClickListener` should return boolean value. It is handler

Reply:
Yeah, thanks for the explanation. I didn't know that `OnClickListener` and `OnLongClickListener` were different in this regard.
https://developer.android.com/reference/android/view/View.OnClickListener
https://developer.android.com/reference/android/view/View.OnLongClickListener